src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceDao.java

```diff
@@ -371,7 +374,13 @@ private GoogleBucketResource retrieveBucketBy(String sql, MapSqlParameterSource
                     .region(region);
             });
 
+        if (googleResourceConfiguration.getAllowReuseExistingBuckets())
```

Comment:
Should this be removed?

Reply:
This is the fix for dev.

Reply:
OH yes

Reply:
Coding and making dinner at the same time is definitely interesting